src/sdk/object_sdk.js

```diff
@@ -435,7 +435,11 @@ class ObjectSDK {
             params.xattr = source_md.xattr;
             params.content_type = source_md.content_type;
         }
-        if (params.xattr) params.xattr = _.omitBy(params.xattr, (val, name) => name.startsWith('noobaa-namespace'));
+        try {
+            if (params.xattr) params.xattr = _.omitBy(params.xattr, (val, name) => name.startsWith('noobaa-namespace'));
+        } catch (e) {
+            dbg.log3("Got an error while trying to omitBy param.xattr:", params.xattr, "error:", e);
```

Comment:
@liranmauda Why dbg3? is this an expected situation? I thought the reason you wanted to add try-catch was to be able to figure out why we have this problem by printing the xattr, or do you already know why it fails?

Reply:
We can divide it into 2:
1. In this case we are trying to remove names that start with `noobaa-namespace` as we don't have a name at all, This is pointless, will fail, and fail the whole copy with it. Ignoring this failure as we don't need to repopulate the xatter in cases that there is no name field. 
2. The printing is indeed for knowing what is in the xatter, and If we need to debug we will have it.

Combining the 2, this is the reason for setting dbg3. We want to avoid this kind of issue, but we might want to print what's in for further debug. 
As for the reason why in some cases the xatter is without a name, we could not see it yet, and we might want to check the `aws sync` command and see why it was sent like this.

Reply:
I don't understand what you mean when you say xattr has no name. We are not looking for a name field, instead we use omitBy to remove any keys that start with this prefix. How can we have an entry in xattr that doesn't have a key?